File: C Programms/while loop/factorial.c
//WAP to print the factorial of a no.
#include<stdio.h>
void main()
{
    int n,i=1,f=1;
    printf("Enter the no. = ");
    scanf("%d",&n);
    while(i<=n)
        f*=i++;
    printf("Factorial of %d is %d ",n,f);
}

File: JAVA/Number check/Factorial.java
import java.util.Scanner;
class Factorial
{
    public static void main(String as[])
    {
        int f=1,i,n;
        Scanner ob=new Scanner(System.in);
        System.out.print("Enter the no.=");
        n=ob.nextInt();
        for(i=1;i<=n;i++)
        {
            f*=i;
        }
        System.out.print(""+f);
    }
}